app/src/main/java/com/example/musicapp/dao/MusicAppDAO.kt

```diff
@@ -3,14 +3,12 @@ package com.example.musicapp.dao
 import com.example.musicapp.dto.Instrument
 import retrofit2.Call
 import retrofit2.http.GET
-import retrofit2.http.Query
 
 interface MusicAppDAO {
-    //https://jsonkeeper.com/b/JIH2
-    //json to be updated in the future with planned instruments
+    /**https://jsonkeeper.com/b/JIH2
+     *json to be updated in the future with planned instruments
+     * Gets the list of instruments through retrofit
+     */
```

Comment:
Tidied up comment section.  Here is the documentation comment organization: https://kotlinlang.org/docs/reference/coding-conventions.html#documentation-comments

Reply:
One more change I recommend... since this is KDoc, it belongs above a class (interface) or method, not below.  Move this above the "interface MusicAppDAO {" line.